.travis.yml

```diff
@@ -1,61 +1,45 @@
 language: python
 python: 3.6.6
 
-env:
-  global:
-    - TRAVIS_NODE_VERSION=8.9.4
-    - PGPORT=5433
-
-dist: trusty
-sudo: true
+sudo: required
 
 addons:
-  hosts:
-    - warehouse.local
-  postgresql: 10
   apt:
     packages:
-    - postgresql-10
-    - postgresql-client-10
-    - nasm
+      - docker-ce
```

Comment:
You don't need this. Docker is installed in Travis CI by default.

Reply:
This updates docker to the latest version.